CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Demo] Add Tensorflow BodyPix segmentation demo for `VideoProcessor`.
 
 ### Changed
-
+- Add check for audio host URL during join flow.
```

Comment:
Need a clearer CHANGELOG entry for what we intend to do here.

Reply:
added more details.